Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -350,13 +395,13 @@ at which point this block turns back to a PID controller but with tuned PID para
 <b>Note:</b> If an autotuning is ongoing, i.e., <code>inTunPro.y = true</code>,
 a request for performing autotuning will be ignored.
 </p>
-<h4>Guidance for settings the parameters</h4>
+<h4>Guidance for setting the parameters</h4>
```

Comment:
so do we also need to set the `yHig` to a reasonable value?

Reply:
Yes, I updated the description accordingly.